src/main/java/com/synopsys/integration/detect/tool/detector/inspectors/nuget/OnlineNugetInspectorLocator.java

```diff
@@ -27,36 +27,40 @@
 
 import com.synopsys.integration.detect.workflow.file.DirectoryManager;
 import com.synopsys.integration.detectable.detectable.exception.DetectableException;
+import com.synopsys.integration.function.ThrowingBiFunction;
 
 public class OnlineNugetInspectorLocator implements NugetInspectorLocator {
     private final NugetInspectorInstaller nugetInspectorInstaller;
     private final DirectoryManager directoryManager;
     private final String overrideVersion;
```

Comment:
```suggestion
    @Nullable
    private final String overrideVersion;
```
If overrideVersion is nullable, can you add this annotation?
If it isn't actually nullable, why is it wrapped in `Optional.ofNullable()` below?

Reply:
This was the state I found the source in and was trying to minimize my impact (I believe it's passed through in a couple places). That said, this is definitely a good change and I'll go ahead and make it.